weekly/2023/packit-workshop.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Packit workshop - Fedora Community Edition
+date: 2023-12-05
+authors: mmassari
+tags:
+  - 2023-December
+  - 2023
+  - December
+---
+
+# Packit workshop - Fedora Community Edition
+
+We, the packit team, are happy to invite you to our upcoming workshop about pulling upstream releases to Fedora using Packit.
+
+**When: Wed, Jan 10, 9:30 AM - Wed, Jan 10, 12:30 PM (GMT+1)**
+
+**Where: Online!**
+
+**Who should attend:** Anyone who maintains a package in Fedora or EPEL and would like to automate syncing the upstream releases for it. The automation is most suitable for simple packages with straightforward update processes (e.g. without patches, or need to build in side-tags). It works without access to the upstream repository.
+
+**No package?** It’s not an issue. The workshop can help you understand the Fedora release process and the services involved. During the workshop, you can help someone else automate their package or ask someone around if you can help with the maintenance (that’s usually very welcome!). You can also bring an orphaned package back to life!
+
+**Can’t make it this time?** You can also follow our documentation and ask for help in case of any issue: #packit:fedora.im (Matrix) / #team-packit (Slack).
```

Comment:
```suggestion
**Can’t make it this time?** You can also follow our documentation and ask for help in case of any issue: #packit:fedora.im (Matrix).
```

This is probably not relevant (and useless without knowing the server) for public audience.